bigchaindb_driver/driver.py

```diff
@@ -63,11 +64,56 @@ def outputs(self):
         """
         return self._outputs
 
+    def info(self, headers=None):
+        """Retrieves information of the node being connected to via the
+        root endpoint ``'/'``.
+
+        Args:
+            headers (dict): Optional headers to pass to the request.
+
+        Returns:
+            dict: Details of the node that this instance is connected
+            to. Some information that may be interesting:
+
+                * the server version,
+                * the public key of the node, and
+                * its key ring (list of public keys of the nodes this
+                  node is connected to).
+
+        Note:
+            Currently limited to one node, and will be expanded to
+            return information for each node that the instance this
+            connected to.
```

Comment:
AFAIK, we do not have a way to connect to all the instances in the federation. Is there one?
I also thought that to connect to all the instances, the driver would need to be provided with an explicit list of instances.